PVTPackage/source/MultiphaseSystem/PhaseSplitModel/DeadOilFlash.cpp

```diff
@@ -56,21 +83,38 @@ bool DeadOilFlash::computeEquilibrium( DeadOilFlashMultiphaseSystemProperties &
 {
   const auto & pressure = sysProps.getPressure();
 
-  const DeadOil_PhaseModel & oilPhaseModel = getOilPhaseModel();
-  const DeadOil_PhaseModel & gasPhaseModel = getGasPhaseModel();
-  const BlackOil_WaterModel & waterPhaseModel = getWaterPhaseModel();
+  bool const containsGas = std::find( sysProps.getPhases().cbegin(),
+                                      sysProps.getPhases().cend(),
+                                      pvt::PHASE_TYPE::GAS ) != sysProps.getPhases().cend();
+  bool const containsWater = std::find( sysProps.getPhases().cbegin(),
+                                        sysProps.getPhases().cend(),
+                                        pvt::PHASE_TYPE::LIQUID_WATER_RICH ) != sysProps.getPhases().cend();
+
+  if( containsGas == containsWater )
+  {
 
+  }
```

Comment:
Oops?

Reply:
Sorry I got confused here, I forgot to remove this. It is removed now.